testing/fbgrab/APKBUILD

```diff
@@ -0,0 +1,22 @@
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+pkgname=fbgrab
+pkgver=1.3
+pkgrel=0
+pkgdesc="Framebuffer screenshot program"
+url="https://fbgrab.monells.se"
+arch="all"
+license="GPL-2.0-only"
+depends="zlib libpng libpng-dev"
```

Comment:
that looks definitively  wrong.

`zlib-dev` and `libpng-dev` should be in `makedepends`

Reply:
Also no  `zlib` and `libpng`  in `depends`